pkg/resource/deploy/builtins.go

```diff
@@ -144,6 +146,7 @@ func (p *builtinProvider) Read(urn resource.URN, id resource.ID,
 
 const readStackOutputs = "pulumi:pulumi:readStackOutputs"
 const readStackResourceOutputs = "pulumi:pulumi:readStackResourceOutputs"
+const readStackResource = "pulumi:pulumi:readStackResource"
```

Comment:
def would want to comment these healthily.  i know i got majorly confused at one point between readStackOutputs and readStackResourceOutputs.   however, i do like just having these be implemented with a special intrinsic function

Reply:
Agreed - will do before merging.